components/Cards/cards.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="card.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined" rel="stylesheet">
+</head>
+
+<body>
+    <h1>Vertical Card</h1>
+    <h3>
+        Cards are used to show the data which is related.
+    </h3>
+    <div class="vertical-card">
+        <img src="tshirt.jpg" alt="">
+        <div class="text">
+            <div>Gorgeous White Shirt</div>
+            <p>RS.986 20%off</p>
+        </div>
+        <div class="badge">
+            <span class="material-icons-outlined">favorite</span>
+        </div>
+        <div class="card-btn">
+            <button class="cart">ADDToCart</button>
+            <button class="whishlist">BuyNow</button>
+        </div>
+    </div>
+    </div>
+    <hr>
+    <h1>Horizontal Card</h1>
+    <div class="horizontal-card">
+        <img src="tshirt.jpg" alt="">
+        <div class="horizontal-card-text">
+            <h2>Gorgeous White Shirt</h2>
+            <h4>RS.986 20%off</h4>
+            <div class="button">
```

Comment:
Rs 986 and 20% off looks like together .Please give some more padding between them.

Reply:
okkkk thanks for your suggestion